docs/classes/lsp3-universal-profile.md

```diff
@@ -22,15 +22,15 @@ Deploys and **configures** a [Universal Profile](../../../standards/universal-pr
 Then, it will:
 
 - upload metadata to IPFS and set the [LSP3 Universal Profile](../../../standards/universal-profile/lsp3-universal-profile-metadata) metadata.
-- attach the Universal Receiver Delegate to the LSP0 ERC725 Account.
+- attach the Universal Receiver Delegate to the ERC725 Account contract.
```

Comment:
Maybe it would be good to keep the mention of LSP0 (in parentheses at the end maybe?).
Otherwise, readers might be confused and look at the contract in the ERC725 repository. But this contract is actually defined in the `@lukso/lsp-smart-contracts`.